docs/source/start.ipynb

```diff
@@ -135,6 +135,24 @@
     "automl.search()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If you would like to see real time updates as `AutoMLSearch` runs, you can use the argument `verbose=True`."
```

Comment:
Should we update our docs examples to all use `verbose=True` by default, and then have this section say "If you'd like to suppress stdout output, turn off verbose" ?

Reply:
I like this idea, could be helpful in the doc examples to be loud about the steps that we're taking!